src/modules/blockchain/implementation/web3-service.js

```diff
@@ -609,6 +628,10 @@ class Web3Service {
 
         gasLimit = gasLimit ?? this.convertToWei(900, 'kwei');
```

Comment:
Maybe since estimate gas is working now we shouold remove the 900 kwei default and replace it with some specific default value for each transaction type. We had that constant before gas estimation was working

Reply:
I agree, focus for this PR is transaction prioritization and mitigation of the issue with estimate gas. We have this update for default gas limit in our task list :-)